llvm/lib/CheerpWriter/CheerpWriter.cpp

```diff
@@ -1041,6 +1041,7 @@ CheerpWriter::COMPILE_INSTRUCTION_FEEDBACK CheerpWriter::handleBuiltinCall(const
 			stream << "import(" << threadingObject << ".script).then(m=>m.default()).catch(e=>{if(e!=='LeakUtilityThread')throw e;});";
 		else
 			stream << "importScripts(" << threadingObject << ".script);";
+		stream << "postMessage('hi');";
```

Comment:
is this needed? do we listen for this message on the other side?

Reply:
If required please let's use a more appropriate string. This feel amateurish.

Reply:
This now sends a null instead.